src/components/character/InventoryManager.tsx

```
@@ -287,6 +289,31 @@ const InventoryManager = ({
+                                  )?.features?.map((feature, featIndex) => (
+                                    <li key={featIndex}>
+                                      <span className="font-bold">
+                                        {feature.name}:{" "}
+                                      </span>
+                                      {feature.description}
+                                    </li>
+                                  ))}
+                                </ul>
+                              </div>
+                            )}
+                          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)